tests/SendGrid.Tests/SendGrid.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <TargetFramework>netcoreapp1.1</TargetFramework>

```

Comment:
IIRC there's another PR open which changes this as well. Of course, it's nonsense that this is still at 1.0, but upgrading it to 2.2 or (preferably) 3.0 would make more sense than 1.1.

Reply:
Sorry this was'nt meant to be included, for some reason i was fidding with this project, this can be excluded when i send a PR again